DEPENDENCIES

```diff
@@ -157,9 +157,15 @@ Runtime dependencies:
     License name: MIT License
     Notice:       Copyright (c) 2011-2016 Twitter, Inc.
   
-  ch.qos.logback:1.2.3: 
+  ch.qos.logback:0.1.5: 
```

Comment:
Again this new reference to an old version

Reply:
Looking at mvn dependency:tree
[INFO] +- org.apache.karaf.features:framework:kar:4.3.8:compile
[INFO] |  +- org.ops4j.pax.logging:pax-logging-logback:jar:2.0.18:compile
[INFO] |  |  +- ch.qos.logback:logback-core:jar:1.2.9:compile
[INFO] |  |  +- ch.qos.logback:logback-classic:jar:1.2.9:compile
[INFO] |  |  +- ch.qos.logback.contrib:logback-jackson:jar:0.1.5:compile
[INFO] |  |  +- ch.qos.logback.contrib:logback-json-core:jar:0.1.5:compile
[INFO] |  |  \- ch.qos.logback.contrib:logback-json-classic:jar:0.1.5:compile

It seems to be a different artifact with it's own versioning